packages/orbit-components/src/Skeleton/Svg.js

```diff
@@ -55,65 +55,70 @@ const Rows = ({ count, height, offset, rowBorderRadius }) => {
 
 const Svg = ({
   animate = true,
-  ariaLabelledby,
   children,
   rowBorderRadius = 3,
   rowHeight = 21,
   rowOffset = 20,
   rows = 1,
-  title,
+  title = "loading",
   viewBox,
   ...props
 }: Props): React.Node => {
+  const [loaded, setLoaded] = React.useState(false);
   const [calculatedHeight, setCalculatedHeight] = React.useState(0);
   const { rtl, orbit } = useTheme();
   const duration = `${2}s`;
   const interval = `${0.5}s`;
-  const uid = useRandomId();
-  const id = ariaLabelledby || uid;
-  const idClip = `${id}-clip`;
+  const id = useRandomIdSeed();
+  const idClip = `${id("clip")}-clip`;
+  const titleId = id(title);
```

Comment:
I don't think there's any need for `id` to change when `title` changes.
```suggestion
  const titleId = id("title");
```